frontend/src/components/Common/SimpleModal.tsx

```diff
@@ -0,0 +1,58 @@
+import {
```

Comment:
I believe this is a ConfirmationModal.  "Simple" is too broad of a term. I'm not sure in what way this is "simple"

Reply:
Its simple in that its a modal compenent that you can pass jsx. I feel like "ConfirmationModal" assumes that the modal is only used for confirmation. It could be used any time you need a modal.